docs/credentials-with-issuer-dependent-terms.json

```diff
@@ -89,7 +89,7 @@
   },
   {
     "type": "MillTestReportCredential",
-    "count": 187
+    "count": 191
```

Comment:
@nissimsan just a warning about this number going up : /

Reply:
Oh I know... Looking at it all day. Very sad! :(